src/mococrw/openssl_wrap.h

```diff
@@ -77,6 +77,18 @@ struct SSLDeleter
     }
 };
 
+template <class P, int(Func)(P*)>
+struct SSLIntDeleter
```

Comment:
What is the purpose of this copy of SSLDeleter?

Reply:
Hm, wouldn't this just work if we remove the function pointer primitive and just say `typename Func` here?

Reply:
Done.